job-dsls/jobs/prod/prod_kogito_properties_generator.groovy

```diff
@@ -72,8 +72,8 @@ pipelineJob("${folderPath}/kogito-properties-generator") {
     parameters {
         booleanParam("IS_RELEASE", true, "it defines if the properties file is for prod or not")
         stringParam("BRANCH_NAME", "main", "the branch the nightly was triggered for")
-        stringParam("REPO_URL", "http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/repositories/kogito-main-nightly", "Prod possibility is http://download.devel.redhat.com/rcm-guest/staging")
-        stringParam("DELIVERABLE_REPO_URL", "http://download.devel.redhat.com/devel/candidates")
+        stringParam("REPO_URL", "\${env.BXMS_QE_NEXUS}/content/repositories/kogito-main-nightly", "Prod possibility is \${env.STAGING_SERVER_URL}")
+        stringParam("DELIVERABLE_REPO_URL", "\${env.DOWNLOAD_CANDIDATES}")
```

Comment:
env variables can't be used here, please move this value inside the job itself

Reply:
according to https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/custom/job/emingora/job/envtest/2/console it can be used, so I agree on leave it as it is.